pkg/cmd/pulumi/history.go

```diff
@@ -41,10 +32,11 @@ func newHistoryCmd() *cobra.Command {
 		Use:        "history",
 		Aliases:    []string{"hist"},
 		SuggestFor: []string{"updates"},
-		Short:      "[PREVIEW] Update history for a stack",
-		Long: `Update history for a stack
-
-This command lists data about previous updates for a stack.`,
+		Short:      "[DEPRECATED] Display history for a stack",
+		Long: "Display history for a stack.\n\n" +
+			"This command displays data about previous updates for a stack.\n\n" +
+			"This command is now DEPRECATED, please use `pulumi stack history`.\n" +
+			"The command will be removed in Pulumi v3.0.0",
```

Comment:
Should we be more vague about timeframe?

```suggestion
			"The command will be removed in a future release.",
```

Reply:
Agreed!